src/app/user-profile/user-info/user-info.component.html

```diff
@@ -0,0 +1,64 @@
+<div class="flex-end button-wrapper">
+  <app-button className="icon-button">
+    <span class="icon-button__plus"> </span>
+  </app-button>
+  <app-button className="icon-button">
+    <span class="icon-button__edit"></span>
+  </app-button>
+</div>
+
+<div class="user-wrapper">
+  <div class="flex-center user">
+    <div class="flex user__info">
+      <div class="user__info-image">
+        <img
+          class="user-image"
+          src="../../../assets/images/user-image.png"
+          alt="user name"
+        />
+      </div>
+
+      <div class="user__info-name">
+        <h2 class="user-name">
+          {{ loggedInUser?.firstName + " " + loggedInUser?.lastName }}
+        </h2>
+        <div class="user-occupation">Collaborator & Editor</div>
+      </div>
+    </div>
+
+    <div class="user__description">
+      {{ userDescription }}
+    </div>
+
+    <div class="flex user__contact">
+      <a class="flex-center user__contact-icon">
+        <img
+          class="icon-facebook"
+          src="../../../assets/icons/facebook-icon.svg"
+          alt="facebook icon"
+        />
+      </a>
+      <a class="flex-center user__contact-icon">
+        <img
+          class="icon-twitter"
+          src="../../../assets/icons/twitter-icon.svg"
```

Comment:
store this value to a variable and then use that variable with src attribute as a property binding.